k8s-readwrite.tf

```diff
@@ -82,6 +82,10 @@ resource "kubernetes_deployment" "apiary_hms_readwrite" {
             name  = "SNS_ARN"
             value = var.enable_metadata_events == "" ? "" : join("", aws_sns_topic.apiary_metadata_events.*.arn)
           }
+          env {
+            name  = "ENABLE_ATLAS_HIVE_BRIDGE"
```

Comment:
Since you also changed the name of the env var passed to the docker image, are you also going to update https://github.com/ExpediaGroup/apiary-metastore-docker/blob/feature/EGDL-152/files/startup.sh#L87?  If not, I don't think this will work.

Reply:
Yes i have updated it already, there are a lot of changes on docker-metastore side so will create a PR for it shortly